apps/server/userInformation.ts

```diff
@@ -0,0 +1,36 @@
+import { APIGatewayProxyHandler } from "aws-lambda";
+import { ServerlessMysql } from "serverless-mysql";
+
+const mysql: ServerlessMysql = require("serverless-mysql")({
+    config: {
+        host: process.env.ENDPOINT,
+        database: process.env.DATABASE,
+        user: process.env.USER,
+        password: process.env.PASSWORD
+    },
+    library: require( "mysql2" )
+})
+
+interface UserInfo {
+    fullName: string
+    mobileNumber: string
+    emailAddress: string
+    brand_Newsletter: boolean
+    ulp_Newsletter: boolean
+}
+
+export const userInformation: APIGatewayProxyHandler = async (event) => {
+    const body = JSON.parse(event.body ?? "") as UserInfo
```

Comment:
Can we make it so if `body` is blank (after parsing), we return `500` instead?

Reply:
And maybe wrap the whole block with `try { ... } catch` just to handle errors properly